src/aura/HH_AddressMgr/HH_AddressMgr.cmp

```diff
@@ -0,0 +1,159 @@
+<aura:component >
+    <!-- public attributes -->
+    <aura:attribute name="listAddr" type="Address__c[]" access="public" />
+
+    <!-- private attributes -->
+    <aura:attribute name="addrDefault" type="Address__c" access="private" />
+    <aura:attribute name="addrNew" type="Address__c" access="private" />
+    <aura:attribute name="showChangeAddressPopup" type="Boolean" access="private" />
+    <aura:attribute name="iAddrSelected" type="Integer" default="0" access="private" />
+    <aura:attribute name="isExistingAddrSectionOpen" type="Boolean" default="true" access="private" />
+    
+    <!-- events we handle -->
+    <aura:handler name="init" value="{!this}" action="{!c.doInit}" />
+    <aura:handler name="change" value="{!v.listAddr}" action="{!c.onChangeListAddr}" />
+
+    <!-- events we fire -->
+   	<aura:registerEvent name="AddressChangedEvent" type="c:HH_AddressChangedEvent"/>
+    
+    <!-- markup follows... -->
+    <div class="slds-grid slds-p-around--x-small">
+        <div class="slds-card slds-p-around--none">
+            <div class="slds-card__body slds-m-around--x-small slds-p-vertical--none" >
+                <!-- address -->
+                <aura:if isTrue="{!v.addrDefault != null}">
+                    <div class="slds-text-body--small" >
+                        <ui:outputText value="{!v.addrDefault.MailingStreet__c + 
+                        	(v.addrDefault.MailingStreet2__c != null ? '\r' + v.addrDefault.MailingStreet2__c : '')}" /><br/>
+                        <ui:outputText value="{!v.addrDefault.MailingCity__c + (v.addrDefault.MailingState__c != null ? ', ' : '') + v.addrDefault.MailingState__c + ' ' + v.addrDefault.MailingPostalCode__c + ' '}" /><br/>
+                        <ui:outputText value="{!v.addrDefault.MailingCountry__c}" />                                     
+                    </div>
+                </aura:if>
+            </div> 
+            <div class="slds-card__footer">
+                <div class="slds-grid slds-grid--align-center">
+                    <ui:button class="slds-button slds-button--small slds-button--neutral" 
+                               aura:id="btnChangeAddress" 
+                               buttonTitle="Change address" 
+                               label="Change address" 
```

Comment:
Do we have or should we have a story to go back and update all strings to use custom labels in the manage household components?

Part of that task will be having to figure out how to deal with the namespace resolution issue with LC labels, unfortunately.